Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.9-alpine AS build
+
+RUN apk add --update zstd-static zstd-dev make gcc musl-dev git
+RUN go get golang.org/x/lint/golint
+RUN mkdir -p /go/src/github.com/Vertamedia/chproxy
+WORKDIR /go/src/github.com/Vertamedia/chproxy
+COPY . ./
+RUN make release-build
+
+FROM alpine
```

Comment:
Why not `scratch`? See https://blog.codeship.com/building-minimal-docker-containers-for-go-applications/ . This would reduce image size.

Reply:
I tried, but the `zstd` package uses CGO, and I couldn't make it build with the static library.
Since the image weighs in at less than 16M, I didn't want to spend more time fighting with CGO to figure out why it didn't want to build (its error messages leave a lot to be desired).

Reply:
OK, then let's start with `alpine` base image :)